core/domain/summary_services.py

```diff
@@ -131,3 +131,18 @@ def _generate_query(categories):
         })
 
     return results
+
+
+def get_featured_explorations():
+    """Returns a list of featured explorations."""
+    exp_ids = []
+    exp_ids = exp_services.get_all_exploration_summaries()
+    exp_summaries = get_displayable_exp_summary_dicts_matching_ids(
```

Comment:
The name for this should be exp_summary_dicts, to distinguish them from the exploration summary domain object.